drizzlepac/haputils/astrometric_utils.py

```diff
@@ -577,15 +572,13 @@ def build_auto_kernel(imgarr, whtarr, fwhm=3.0, threshold=None, source_box=7,
     kern_img[:, -edge:] = 0.0
     kernel_psf = False
 
-    peaks = find_peaks(kern_img, threshold=threshold * 5,
-                       box_size=isolation_size)
+    peaks = photutils.detection.find_peaks(kern_img, threshold=threshold * 5, box_size=isolation_size)
```

Comment:
With the change in import statements for the photutils functions, this line only needs 'peaks = find_peaks('.

Reply:
issue resolved by commit [8892775](https://github.com/spacetelescope/drizzlepac/pull/986/commits/8892775dfdbf046c425976b4d1d831517eae30bf)